frontend/src/components/RuntimeCheck.tsx

```
@@ -0,0 +1,48 @@
+            <div className={styles.userInput}>
+                <select value={userSelection} onChange={handleChange}>
+                    <option value="">Select the runtime</option>
+                    <option value="O(1)">O(1)</option>
+                    <option value="O(log n)">O(log n)</option>
+                    <option value="O(n)">O(n)</option>
+                    <option value="O(n log n)">O(n log n)</option>
+                    <option value="O(n^2)">O(n^2)</option>
+                    <option value="O(n^3)">O(n^3)</option>
+                    <option value="O(2^n)">O(2^n)</option>
+                </select>
+                <button onClick={handleCheck} disabled={!userSelection}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)